cep-20.2.md

```diff
@@ -0,0 +1,650 @@
+    preserve_egg_dir: bool (default false)
+
+    # only used as a hack to down-prioritize packages
+    track_features: [string]
+
+    # skip compiling pyc for some files
+    skip_compile_pyc: [glob]
+
+    # do not
+    no_link: [glob]
+
+    # the script that is executed to build the package
+    # if it is only one element and ends with `.sh` or `.bat`
+    script: string | [string]
+
+    # linux only, list of rpaths
+    rpaths: [path] (defaults to ['lib/'])
+
+    # include files even if they are already in the environment as part of some other
+    # host dependency
+    always_include_files: [path]
+
+    # wether to relocate binaries or not
+    binary_relocation: bool (defaults to true)
+
+    # force file to be detected as TEXT file (for prefix replacement)
+    has_prefix_files: [path]
+
+    # force file to be detected as BINARY file (for prefix replacement)
+    binary_has_prefix_files: [path]
+
+    # ignore all or specific files for prefix replacement
+    ignore_prefix_files: bool | [path] (defaults to false)
+
+    # wether to detect binary files with prefix or not
+    detect_binary_files_with_prefix: bool (defaults to true)
+
+
+    # Wether to include the recipe or not in the final package
+    include_recipe: bool (defaults to true)
+
+    # environment variables to either pass through to the script environment or set
+    script_env: [env_vars]
+
+    run_exports: [MatchSpec] OR {strong: [MatchSpec], weak: [MatchSpec], strong_constrains: [MatchSpec], weak_constrains: [MatchSpec], noarch: [MatchSpec]}
+
+    missing_dso_whitelist: [glob]
+    runpath_whitelist: [glob]
+
+    # This is only used in the pip feedstock.
+    disable_pip: bool (defaults to false)
+
+    # used in many r packages on Windows
+    merge_build_host: bool
+
+    # ignore run exports by name
+    ignore_run_exports: list
+
+    # ignore run exports coming from the specified packages
+    ignore_run_exports_from: list
+
+    # copies fn[.bat/.sh] to the appropriate location, adds `.bat` or `.sh` to the filename
+    pre-link: string
+    post-link: string
+    pre-unlink: string
+
+    # error out when overdepending
+    error_overdepending: bool (defaults to ?)
+    # error out when overlinking
+    error_overlinking: bool (defaults to ?)
+
+    # REMOVED:
+    # noarch_python: bool
+    # features: list
+    # msvc_compiler: str
+    # requires_features: dict
+    # provides_features: dict
+    # preferred_env: str
+    # preferred_env_executable_paths: list
+
+    # marked as "still experimental"
+    # pin_depends: Enum<"record" | "strict">
+    # overlinking_ignore_patterns: [glob]
+
+    # defaults to patchelf (only cudatoolkit is using `lief` for some reason)
+    # rpaths_patcher: None
+```
+
+## Source section
+
+```yaml
+source: [SourceElement]
+```
+
+where the different source elements are defined as follows.
+
+### URL source
+
+```yaml
+# url pointing to the source tar.gz|zip|tar.bz2|...
+url: url
```

Comment:
This can be a list of URLs too (mirrors of the same package with the same hash; used by R packages)

Reply:
👍